src/manifest.json

```diff
@@ -1,15 +1,21 @@
 {
-  "name": "RocketChat",
-  "short_name": "RocketChat",
+  "name": "Rocket.Chat",
+  "short_name": "Rocket.Chat",
   "start_url": ".",
   "display": "fullscreen",
   "background_color": "#04436a",
+  "theme_color": "#04436a",
   "description": "Amazing chat app",
   "orientation": "portrait-primary",
   "icons": [
     {
-      "src": "assets/images/rocketchat-bubble.png",
```

Comment:
but you didnt mention size 144X144 here only 192, 384. dont you need to explicitly tell the browser that an image can be used as 144*144 image?